### PR TITLE
New API names (e.g. tp.input_node, evset.node, to.to_pandas)

### PR DESCRIPTION
**Pull request by achoum**
Based on: https://github.com/google/temporian/pull/146

### CODE REVIEW

File: temporian/__init__.py [hunk 1]
```diff
@@ -23,51 +23,76 @@
...
-from temporian.core.data import node
-from temporian.core.data import schema
+# # Load packages privately
+# from temporian.core import serialization as _serialization
+# from temporian.core.data import dtype as _dtype
+# from temporian.core.data import node as _node
+# from temporian.core.data import schema as _schema
+# from temporian.core.data import duration as _duration
+# from temporian.core import evaluation as _evaluation
+
```
Comment (rstz) [inline temporian/__init__.py:33]: Do we need this comment?

**Review by rstz — APPROVED** [review]
LGTM

**Review by rstz — APPROVED** [review]
LGTM

**Comment by achoum** [comment]
Thanks

**Pull request merged**